wagtail/wagtailadmin/views/account.py

```diff
@@ -40,7 +42,8 @@ def account(request):
 
     return render(request, 'wagtailadmin/account/account.html', {
         'show_change_password': password_management_enabled() and request.user.has_usable_password(),
-        'show_notification_preferences': show_notification_preferences
+        'show_notification_preferences': show_notification_preferences,
+        'show_preferred_language_preferences': len(get_available_admin_languages()) > 1
```

Comment:
`bool(get_available_admin_languages())` wouldn't be correct, because we also want the option to be hidden when there is _one_ language available.